CONTRIBUTING.md

```diff
@@ -203,6 +203,40 @@ or
 $ yarn start --disable-host-check --public=$(echo ${server.dev-server} | sed -e s/https:\\/\\/// -e s/http:\\/\\/// -e s/\\///) --host="0.0.0.0" --env.target=${CHE_API_EXTERNAL%????}
 ```
 
+#### Step 7: Testing new workspace loader
+
+When workspace loader server is run, one probably need to test introduced changes.
+
+To safely test the changes it is better to create a separate workspace which will be used by new loader.
+But by default, Che doesn't allow to run more than one workspace simultaneously.
+To change this behaviour one need to set `che.limits.user.workspaces.run.count` Che property to value greater than `1`.
+In development environment that could be reached by adding `CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` environment variable for Che server deployment.
+Please note, after changing deployment config one need to apply changes by rolling out (or rescaling) the corresponding pod 
+(in case of Openshift just add the environment variable via Openshift dashboard in the `Environment` tab of the Che server deployment and the pod will be rolled out automatically).
+
+To be able to point new workspace loader to the test workspace it is required to add the the test workspace id to the path of workspace loader route.
+So, first, we need to retrieve the test workspace id.
+This could be done using swagger (please note, it might be disable on production environment).
```

Comment:
User could grab workspaceId like env variable of  any workspace container, so maybe it is better to apply this information instead of swagger.

Reply:
Yes, this is useful information, however requires to have running workspace. I think it would be better to not to require running state of the test workspace (because in many cases workspace loader should be tested with workspace start flow).